workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx

```diff
@@ -64,12 +68,27 @@ const activityClaimStatusClassName = {
 const ActivityClaimPanel = ({
 	activity,
 	activityIndex,
+	errors,
 	hasPermissionEditClaimActivity,
+	isButtonClicked,
+	isDraft,
 	overallCampaignDescription,
 	setFieldValue,
 }: IProps & Pick<FormikContextType<MDFClaim>, 'setFieldValue'>) => {
 	const [expanded, setExpanded] = useState<boolean>(!activity.selected);
 
+	const isNotBudgetSelected = Array.isArray(errors?.activities)
```

Comment:
would it be possible to invert this variable naming and initialization? like make it named `isBudgetSelected`, and when using it negate it like `!isBudgetSelected`. we don't normally add the negation to the name like `isNot`

Reply:
Ok, sounds good to me!
Done!